plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -19,20 +19,22 @@
     LauncherTestSuite,
 )
 from hydra.test_utils.test_utils import chdir_hydra_root, chdir_plugin_root
-from omegaconf import OmegaConf
 from pytest import fixture, mark
 
-from hydra_plugins.hydra_ray_launcher._launcher_util import (  # type: ignore
-    _run_command,
-    ray_down,
-    ray_new_dir,
-    ray_rsync_up,
-    ray_up,
-)
 from hydra_plugins.hydra_ray_launcher.ray_aws_launcher import (  # type: ignore
     RayAWSLauncher,
 )
 
+# mypy complains about "unused type: ignore comment" on macos
+# workaround adapted from: https://github.com/twisted/twisted/pull/1416
+try:
+    import importlib
+
+    sdk = importlib.import_module("ray.autoscaler.sdk")
```

Comment:
If you change this to:
```python
    sdk : Any = importlib.import_module("ray.autoscaler.sdk")
```

or:
```python
    sdk = importlib.import_module("ray.autoscaler.sdk") # type: ignore
```

Does it eliminate the need for all the specific mypy type comments all over?

Reply:
`sdk : Any = importlib.import_module("ray.autoscaler.sdk")` works on linux and gets rid of the type comments, checking macos now with the CI run. The second one doesn't get rid of the type comments all over and in addition gives a `error: unused 'type: ignore' comment` on the import line.

Reply:
Looks like `sdk : Any = importlib.import_module("ray.autoscaler.sdk")` gets rid of the type comments for both linux and macos.

Reply:
Awesome.

Side note:
I am really curious about the root cause of the difference between Mac and Linux here.

Reply:
There might be some platform specific issues with mypy but may be mypy is not the issue here and the search results sound like what I want to hear. It seems like the main problem is with ray wheels: https://github.com/ray-project/ray/issues/14431. https://github.com/ray-project/ray/pull/14492 tried to fix it but I still could not find the `py.typed` file in the 1.3.0 release. 

The macos wheel does have the py.typed file, and so the normal import succeeds. The actual type errors on macos kinda make sense and is probably due to some type usage (also I'm a total beginner on the whole type system thing in python, so don't really have an in depth understanding unfortunately).